region/wreckedship/main/Wrecked Ship East Super Room.json

```diff
@@ -378,6 +378,105 @@
       },
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "Gain Flash Suit (Spikesuit), Conveyor Run",
+      "requires": [
+        "f_DefeatedPhantoon",
+        {"enemyKill": {
```

Comment:
You can kill them with speed.... maybe insaneJump is good enough.  ~Its tricky to avoid taking damage but you can do it.~ I have a setup, so its fine.  You just might have to kill them 1 by 1 since its hard to get them both at once.  And I guess you lose a trivial amount of energy shinesparking.